src/main/resources/publisher_channel_registry_v2.csv

```diff
@@ -1,3534 +1,4139 @@
+"14D8F0E1-6B89-498C-BB3F-532981B49FBA";"Fountain Publishers"
+"E3AB1B3B-A420-4E9F-8DB8-EA1937ECC647";"Four Courts Press"
+"3E438D33-85D8-4BAD-8000-DE7C6AFFC646";"Frakcija - performing arts journal"
+"8BDD3544-0F76-4BAD-821C-A902DEE808F9";"Frame Publishers"
+"CE2F7ACE-5071-4070-8776-63DDA8F44472";"Franco Cesati"
+"ED71BED3-A722-4958-8A83-07C8C83CBE34";"franco cesati editore"
+"A9536A19-7C31-496D-8F54-AF7E33055078";"Frank & Timme"
+"9BE6D213-540F-4F24-8358-229DCA78C1CF";"Frank Cass Publishers"
+"1642B6A2-F37F-4450-9154-CC934235C49F";"Franz Steiner Verlag"
+"02458251-0606-4B16-BD55-F1991D467EE0";"Fraunhofer IRB Verlag"
+"361501C2-9D83-4466-B5B6-3BC7B2C95E75";"Freedom House"
+"157438CA-6819-46E0-B39F-0A62799A6EBB";"Freidrich-Ebert-Stiftung"
+"E07F8D3F-5A63-43F6-8F12-379DF61412B4";"Freie Universität Berlin"
+"B2E63637-A54C-4EA5-9412-6824703F6325";"Fréttin"
+"816AC546-C899-4371-A4B3-02E64EFAE44E";"Freund Publishing House, Ltd."
+"ED2190BD-B242-4DE6-A374-83A60E246F38";"Freunde der Salzburger Geschichte"
+"8B1A4810-30C4-4FF3-994A-6B55338DB65D";"Freundeskreis Bätter"
+"68F033CA-4085-4C9E-8C52-93B6A48095A1";"Freundeskreis Freilichtmuseum Südbayern e. V. Freundeskreis Blätter, Band 41"
+"8A94AE1F-41C0-4231-B355-597ADB17A073";"Fride"
+"1FD6D853-2987-4882-B0B7-E47FB0C6EFB8";"Fridtjof Nansens Institutt"
+"5D1EE9F4-D8D2-4B00-878B-3195DC75106D";"Friedrich-Ebert-Stiftung"
+"E87795D9-2BC2-4200-81F5-8E85466E7EFA";"Fritzes"
+"FCFF4EC6-502D-4036-A17D-FC2CF98113EC";"Frommann-Holzboog"
+"2574A080-9639-4CF4-B070-94BDA9CE3DA9";"Frontiers in Cell and Developmental Biology"
+"E23408B6-2A77-4B0A-8E55-6BC435EF99E1";"Frontiers Media S.A."
+"DFE2333C-85BD-4885-8350-B668E8CD6B2D";"Frontiers Media S.A."
+"596ED54E-A0D7-4DB1-AB26-D36036AB2EBA";"Frydenlund Academic"
+"9D168CF4-054B-445A-9D99-EB1DB820B6BA";"FTA Publications"
+"1C061346-DF60-4358-8766-00B6C0B7CA3D";"Fudan University Press"
+"81BA97B7-EBB3-4497-88E3-4C5A82E121E9";"FUNCAMP - Fundacao de Desenvolvimento da UNICAMP"
+"2B0017B7-3510-4864-BD21-08093749F71D";"Fundación Biblioteca Virtual Miguel de Cervantes"
+"FC0A8F6C-7E1B-433C-88AD-E5908ECE6E96";"Fundacion Infancia y Aprendizaje"
+"A02B3B26-6C17-4D37-939C-1B2B6B07680F";"Fundación Universitaria Española"
+"24FD807A-F5DE-4780-B8A3-4577DDDD9A51";"Fundación Universitaria Española"
+"FED167AB-6E87-4438-A451-7B47FAE7D0A6";"Fundan Press"
+"DC88E411-C585-4A80-BEEE-1E32DC723E91";"Fungiflora"
+"48BA6E46-926F-41B7-9CDC-C8E44313F6F3";"Fysikaktuellt"
+"A130B35D-A5F0-4F7E-9A2C-4B5DC5A92D9C";"Förlag Göteborgs Universitet"
+"7B25DE1B-A353-4D13-AB84-3DF6218537CF";"Føroya Fródskaparfelag"
+"044E99C3-6463-42DE-A1D6-4DE1FA71E578";"Församlingsförlaget"
+"E3E136BB-001E-4BC0-AFC4-537DFAA63822";"Försvarshögskolan (FHS)"
+"6DBA8228-1349-4B80-BF15-2F4F232BDDED";"G. Giappichelli Editore"
+"F5FE8480-F30F-4545-A13F-39633A83234F";"G.B. Palumbo & C. Editore"
+"369DDA02-E1F0-449F-A593-04B2CBF2A649";"Gabler Verlag"
+"4ED9E934-5E00
```

Comment:
the new file does not have multiple representations of NTNU like "NTNU Vitenskapsmuseet", is it the intention?